src/components/AddBooks/AddBook.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from "react";
+import axios from "axios";
+import Navigation from "../Navigation/Navigation";
+import Grid from "../Grid/Grid";
+
+const AddBooks = () => {
+  const [title, setTitle] = useState("");
+  const [author, setAuthor] = useState("");
+  const [quantity, setQuantity] = useState("");
+  const [search, setSearch] = useState("");
+  const [books, setBooks] = useState([]);
+  const [selectedBook, setSelectedBook] = useState(null);
+  const [orderQuantity, setOrderQuantity] = useState(1);
+  const [orderSuccess, setOrderSuccess] = useState(false);
+  const [orderError, setOrderError] = useState(false);
+
+  const handleTitleChange = (event) => {
+    setTitle(event.target.value);
+  };
+
+  const handleAuthorChange = (event) => {
+    setAuthor(event.target.value);
+  };
+
+  const handleQuantityChange = (event) => {
+    setQuantity(event.target.value);
+  };
```

Comment:
these can all be handled directly in the onChange no need for an extra function just set it `(e) => setTitle(e.target.value)` for example